firebase-installations/src/main/java/com/google/firebase/installations/FirebaseInstallations.java

```diff
@@ -31,6 +31,7 @@
 import com.google.firebase.installations.remote.FirebaseInstallationServiceClient;
 import com.google.firebase.installations.remote.InstallationResponse;
 import com.google.firebase.installations.remote.InstallationResponse.ResponseCode;
+import com.google.firebase.installations.remote.TokenResult;
```

Comment:
Github doesn't allow me to leave comment for unchanged lines, so I leave a comment here for Line#61 in this file:

How about using a more specific name for the variable 'lock' to indicate what the lock is used for?

Reply:
It is a generic lock used in this class for state listeners and boolean variable. Idk what would be the right way to capture the purpose aptly.